src/test/java/uk/gov/hmcts/reform/jobscheduler/controllers/jobscontroller/GetAllTest.java

```diff
@@ -72,6 +105,29 @@ public void should_return_401_when_auth_token_is_invalid() throws Exception {
         sendGet().andExpect(status().isUnauthorized());
     }
 
+    @Test
+    public void should_return_400_when_request_parameters_are_out_of_bounds() throws Exception {
+        sendGet("/jobs?page=-1")
+            .andExpect(status().isBadRequest())
+            .andExpect(jsonPath("error").value("must be greater than or equal to 0"));
+
+        int minSize = JobsController.MIN_PAGE_SIZE - 1;
+
+        sendGet("/jobs?size=" + minSize)
+            .andExpect(status().isBadRequest())
+            .andExpect(jsonPath("error")
+                .value("must be greater than or equal to " + JobsController.MIN_PAGE_SIZE)
+            );
+
+        int maxSize = JobsController.MAX_PAGE_SIZE + 1;
```

Comment:
It's bigger than 'max size'. How about `tooBigPageSize`? (and `tooSmallPageSize` above)

Reply:
semantics. can declare variously: `newMax`, `overMax`, `tooBig`, `maxBreach`.. in combination with `size` and `page` too. the point still clear among couple of lines.

let's say if some1 reads the code it's shorter and more correct:
> Let's declare that maximum size will be this and that then we are going to use it as a new limit in the request parameter...

Reply:
by sampling the code read, can see that probably best way, if necessary, is to expand to `newMaximumPageSize` with the highest count of characters